### PR TITLE
debian: fix symlink issue post install/upgrade

### PR DESCRIPTION
**Pull request by andrijapanicsb**
## Types of changes

- [x ] Cleanup (Code refactoring and cleanup, that may add test cases)

## How Has This Been Tested?

About to be tested!

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
@blueorangutan package

**Comment by rhtyd** [comment]
@blueorangutan package

File: debian/cloudstack-usage.postinst [hunk 1]
```diff
@@ -21,15 +21,10 @@ set -e
...
-        if [ -f "/etc/cloud/management/db.properties" ]; then
-            cp -a /etc/cloud/management/db.properties /etc/cloudstack/usage/db.properties
-        fi
-
-        # Replacing db.properties with management server db.properties
+     
+        # Linking usage server db.properties to management server db.properties
         if [ -f "/etc/cloudstack/management/db.properties" ]; then
-            rm -rf /etc/cloudstack/usage/db.properties
+            rm -f /etc/cloudstack/usage/db.properties
```
Comment (DaanHoogland) [inline debian/cloudstack-usage.postinst]: this would work, but wouldn't ```ln -sf ...``` as well?

File: debian/cloudstack-usage.postinst [hunk 2]
```diff
@@ -40,7 +35,7 @@ case "$1" in
 
         # Replacing key with management server key
         if [ -f "/etc/cloudstack/management/key" ]; then
-            rm -rf /etc/cloudstack/usage/key
+            rm -f /etc/cloudstack/usage/key
             ln -s /etc/cloudstack/management/key /etc/cloudstack/usage/key
```
Comment (DaanHoogland) [inline debian/cloudstack-usage.postinst]: same as above

Comment (rhtyd) [reply debian/cloudstack-usage.postinst]: Yes. I'm not sure if the file being removed may not be a symlink, this approach should work as intended.

Comment (andrijapanicsb) [reply debian/cloudstack-usage.postinst]: KISS, as far as I'm concerned :)

**Comment by andrijapanicsb** [comment]
I've manually tested this one, and it works fine as long as we respect the expectation that the usage server is installed once both the db.properties and the key files are present in the mgmt server folder.

**Pull request merged**